src/main/java/uk/gov/pay/api/model/search/card/RefundForSearchRefundsResult.java

```diff
@@ -63,7 +63,7 @@ public String getChargeId() {
     public Long getAmountSubmitted() {
         return amountSubmitted;
     }
-    @ApiModelProperty(dataType = "uk.gov.pay.api.model.links.PaymentLinksForSearch")
+    @ApiModelProperty(dataType = "uk.gov.pay.api.model.links.RefundLinksForSearch")
```

Comment:
Well spotted, probably need to update `swagger.json` as well?

Reply:
ok it did now using msl 🤷‍♀️